wpimath/src/test/python/kinematics/test_mecanum_drive_odometry3d.py

```diff
@@ -0,0 +1,259 @@
+import pytest
+import math
+import random
+
+from wpimath.kinematics import (
+    MecanumDriveKinematics,
+    MecanumDriveOdometry3d,
+    MecanumDriveWheelPositions,
+    ChassisSpeeds,
+)
+from wpimath.trajectory import TrajectoryGenerator, TrajectoryConfig
+from wpimath.geometry import Translation2d, Rotation2d, Pose3d, Rotation3d, Pose2d
+
+
+@pytest.fixture
+def odometry3d_test():
+    class MecanumDriveOdometry3dTest:
+        def __init__(self):
+            self.m_fl = Translation2d(x=12, y=12)
+            self.m_fr = Translation2d(x=12, y=-12)
+            self.m_bl = Translation2d(x=-12, y=12)
+            self.m_br = Translation2d(x=-12, y=-12)
+
+            self.zero = MecanumDriveWheelPositions()
+
+            self.kinematics = MecanumDriveKinematics(
+                self.m_fl, self.m_fr, self.m_bl, self.m_br
+            )
+            self.odometry = MecanumDriveOdometry3d(
+                self.kinematics, Rotation3d(), self.zero
+            )
+
+    return MecanumDriveOdometry3dTest()
+
+
+def test_initialize(odometry3d_test):
+    odometry = MecanumDriveOdometry3d(
+        kinematics=odometry3d_test.kinematics,
```

Comment:
nit: I think it's obvious what this param is already :)
```suggestion
        odometry3d_test.kinematics,
```